CMakeLists.txt

```diff
@@ -259,6 +259,15 @@ else()
 
   dolphin_compile_definitions(_DEBUG DEBUG_ONLY)
   check_and_add_flag(GGDB -ggdb DEBUG_ONLY)
+
+  if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
Remove the ${} around the variable name to avoid double expansion.

Reply:
This was copied from some other checks in Dolphin, so those should be updated to avoid double-expansion later as well.